src/Place/ReadModel/Duplicate/CleanPlaceName.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\ReadModel\Duplicate;
+
+use CultuurNet\UDB3\Json;
+use CultuurNet\UDB3\Model\ValueObject\Geography\Address;
+
+class CleanPlaceName
+{
+    private const MAX_LENGTH_TITLE = 150;
+
+    public static function transform(Address $address, string $title): string
+    {
+        if (mb_strlen($title) > self::MAX_LENGTH_TITLE) {
+            return '';
+        }
+
+        if (str_contains($title, $address->getStreet()->toString())) {
+            return '';
+        }
+
+        // the goal is to remove as much HTML as we can, if strip_tags misses a tag, we decode the html chars
+        $title = htmlspecialchars_decode(strip_tags($title));
+
+        $title = self::replaceCountryCodes($title);
+
+        // Decode the unicode characters
+        return Json::decode('"' . $title . '"');
+    }
+
+    private static function replaceCountryCodes(string $title): string
+    {
+        $countries = [
```

Comment:
If we maintain a big list like this I would move it to a config file. Otherwise I would only support BE, NL, FR and DE

Reply:
Both are fine for me. In practice I think BE and NL will cover 99% of all places.
should I just shrink the array?